modules/cnv-pxe-booting-with-mac-address.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-pxe-booting-with-mac-address-{context}"]
+= PXE booting with a specified MAC address
+
+PXE booting, or network booting, is supported in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
+.Prerequisites
+
+* A bridge must be configured on all nodes. The bridge must be connected to
+an interface that allows PXE booting.
```

Comment:
yes, the same vlan-related prerequisites apply here, too. (If the user would like to boot from a pxe-enabled VLAN). I think that it would be best to refer to the former prereq:

* A Linux bridge must be connected, as detailed in connecting-resource-bridge-network-{context}
* A PXE server is connected to the same VLAN of the bridge.

Reply:
@dankenigsberg: I added something similar, including a link. Unfortunately, I can't test the link yet (the path will be different than it is on my local build) but please let me know what you think of the rest when you get the chance. thanks :)

And if this is resolved, let me know if you think we can move it to QE Review.